src/sys/setup.cpp

```diff
@@ -157,7 +157,7 @@ bool Setup::existsProgramsHome() {
 
 void Setup::cloneProgramsHome(const std::string& git_url) {
   // We cannot use getProgramsHome() here because it checks for existence
-  Git::clone(git_url, getLodaHome() + "programs");
+  Git::clone(git_url, getLodaHome() + "programs", 20);
 }
```

Comment:
Add a constant for "20" in `Setup`. Use it at: https://github.com/loda-lang/loda-cpp/blob/2e092c0ebcfcc8c86be30b5710a225500cb901d4/src/mine/stats.cpp#L485